src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -36,6 +37,8 @@ function OptionRowLHNData({
     const optionItemRef = useRef<OptionData>();
 
     const shouldDisplayViolations = canUseViolations && ReportUtils.shouldDisplayTransactionThreadViolations(fullReport, transactionViolations, parentReportAction);
+    const isReportOwner = fullReport?.ownerAccountID === getCurrentUserAccountID();
+    const shouldDisplayReportViolations = isReportOwner && ReportUtils.hasReportViolations(reportID);
```

Comment:
Here we check when we should show RBR in LHN but I'm not sure if the condition is correct 🤔 
Shouldn't we check if the current user ID is in `fullReport?.participants`?

Reply:
I actually updated those lines to show RBR for all participants [here](https://github.com/Expensify/App/pull/44139/commits/9cfb57bd277d0ad4b541d826c42623facd062eab)